ui/src/app/auth/auth.guard.spec.ts

```diff
@@ -26,8 +21,10 @@ describe("AuthGuard", () => {
       imports: [HttpClientTestingModule]
     })
     injector = getTestBed()
-    authService = injector.get(AuthService)
+    // authService = injector.get(AuthService)
```

Comment:
Will this be uncommented for a near-future change?

Reply:
Missed it. Fixed.